test/test_dmrg.jl

```diff
@@ -1,5 +1,5 @@
 @eval module $(gensym())
-using ITensors: ITensors, MPO, OpSum, inner, randomMPS, siteinds
+using ITensors: ITensorMPS, ITensors, MPO, OpSum, inner, randomMPS, siteinds
```

Comment:
```suggestion
using ITensors: ITensors, inner
using ITensors.ITensorMPS: ITensorMPS, MPO, OpSum, randomMPS, siteinds
```

Reply:
I guess it isn't so obvious where `OpSum` lives and likely it should be moved out to a different package, but I associate it more with the MPS code.